not-to-release/sources/answers/20111107194805AAdINwt_ans.xml.conllu

```diff
@@ -27,7 +27,7 @@
 # text = using the metro or the air france bus
 1	using	use	VERB	VBG	VerbForm=Ger	0	root	0:root	_
 2	the	the	DET	DT	Definite=Def|PronType=Art	3	det	3:det	_
-3	metro	metro	PROPN	NNP	Number=Sing	1	obj	1:obj	_
+3	metro	metro	NOUN	NN	Number=Sing	1	obj	1:obj	_
```

Comment:
I think this could be PROPN if referring to a specific city's transit service

Reply:
I'm not sure. If you know that the sentence is about the parisian metro, metro could be seen as PROPN, but there is also a metro in Marseille, Rennes, Montréal and so it's just a different word for underground/subway/tube ?

Reply:
Not all cities with a subway call it a metro. But many do. So I think either interpretation is valid here.

Reply:
I was probably guided by the French word "métro" which is used for any kind of subway train system ("le métro londonien", "le métro de Berlin" ...). Maybe in English the word metro is more used for the Parisian one? So let's keep it PROPN here since the context is clearly Paris?